test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorResolverTest.cs

```diff
@@ -297,7 +297,7 @@ public static TheoryData ResolveDirectiveDescriptorsData
                         new []
```

Comment:
nit: update a couple of the higher-level tag helper tests e.g. one of the test Razor input files to use a more complicated pattern than '*'